app/controllers/api/auth/series_controller.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+class Api::Auth::SeriesController < Api::SeriesController
+
+  include ApiAuthenticated
+
+  api_versions :v1
+
+  filter_resources_by :account_id
+
+  represent_with Api::Auth::SeriesRepresenter
+
+  def sorted(res)
+    res.order('updated_at desc')
```

Comment:
this is inconsistent with the non-auth series controller, perhaps we should make them match?